docs/_deploying/heroku.md

```diff
@@ -0,0 +1,280 @@
+
+Now that your code is deployed, we need to start a web server instance using the Heroku CLI:
+
+```shell
+heroku ps:scale web=1
+```
+
+**2. Update your Slack app's setting**
+
+In the [Getting Started guide][getting-started-guide], we used ngrok as your **Request URL** for actions and events. Now we need to use your Heroku web address.
+
+Get your Heroku web address with the following command:
+
+```shell
+heroku info
+# ...
+# Web URL: https://sharp-rain-871.herokuapp.com/
+```
+
+In our example, the web address is `https://sharp-rain-871.herokuapp.com/`.
+
+Now browse to the [Slack App page](https://api.slack.com/apps) and select your app name. Next, we'll update your **Request URL** in two locations to be your web address.
+
+> 💡 Your **Request URL** ends with `/slack/events`, such as `https://sharp-rain-871.herokuapp.com/slack/events`.
+
+First, select **Interativity & Shortcuts** from the side and update the **Request URL**:
+
+![Interactivity & Shortcuts page](../assets/interactivity-and-shortcuts-page.png "Interactivity & Shortcuts page")
+
+Second, select **Event Subscriptions** from the side and update the **Request URL**:
+
+![Event Subscriptions page](../assets/event-subscriptions-page.png "Event Subscriptions page")
+
+> 💡 Free Heroku apps sleep when inactive. 💤 If your verification fails, please try it again immediately.
+
+**3. Test your Slack app**
+
+Your app is now deployed and Slack is updated, so let's try it out!
+
+Open a Slack channel that your app has joined and say "hello" (lower-case). Just like in the [Getting Started guide](/bolt-js/tutorial/getting-started#sending-and-responding-to-actions), your app should respond back.
+
+---
+
+### Deploy an update
+
+As you continue building your Slack app, you'll need to deploy updates. A common flow is to make a change, commit it, and then push it to Heroku.
+
+<!-- TODO - Update GitHub link -->
+Let's get a feel for this by updating your app to respond to a "goodbye" message. Add the following code to `app.js` ([source code on GitHub](https://github.com/mwbrooks/bolt-js-getting-started-app/blob/master/app.js)):
+
+```javascript
+// Listens to incoming messages that contain "goodbye"
+app.message('goodbye', async ({ message, say }) => {
+  // say() sends a message to the channel where the event was triggered
+  await say(`See ya later, <@${message.user}> :wave:`);
+});
+```
+
+Commit the changes to your local Git respository:
+
+```shell
+git commit -am "Say 'goodbye' to a person"
+```
+
+Deploy the update by pushing to your `heroku` remote:
+
+```shell
+git push heroku main
+```
+
+When the deploy is complete, you can open a Slack channel that your app has joined and say "goodbye" (lower-case). You should see a friendly farewell from your Slack app.
+
+---
+
+### Next steps
+
+You just deployed your first ⚡️[Bolt for JavaScript app to Heroku](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app)! 🚀
```

Comment:
I'll open a team discussion to talk about where the example apps should live - separate repository, subfolder in bolt, or elsewhere. Afterwards, I'll update the links in this guide.

Reply:
I think we can have it in a subfolder in bolt or as a separate repo in github.com/slackapi

Reply:
Added the _Getting Started with Heroku_ app to the `examples/` directory in commit e34e905